### ISSUE TITLE
Floodfill issue

### ISSUE THREAD

**Issue by Champ-007** [Null-MC/Shrimple#93]
composite12.fsh: composite12.fsh: 0(1168) : error C1503: undefined variable "GetLPVPosition" 0(1169) : error C1503: undefined variable "SceneLPVSize" 0(1170) : error C1503: undefined variable "SampleLpv" 0(1171) : error C1503: undefined variable "GetLpvFade" 0(1173) : error C1503: undefined variable "GetLpvBlockLight"
Issue with transitioning to using floodfill. Though I think this may have been caused by not setting the "size" attribute off of "none" to something like "small"

**Comment by null511** [comment]
yep, if you manually enabled floodfill and left the buffer off/zero, then it'll throw an error. I keep forgetting to fix that
